apps/web/test/lib/next-config.test.ts

```diff
@@ -1,173 +1,179 @@
+    
+      orgUserRouteMatch:pathToRegexp(orgUserRoutePath)
+    }
   })
+});
 
+describe('next.config.js - RegExp', ()=>{
 	it("Booking Urls", async () => {
-		expect(userTypeRouteRegExp.exec('/free/30')?.groups).toContain({
+		expect(userTypeRouteMatch('/free/30')?.params).toContain({
 			user: 'free',
 			type: '30'
 		})
 		
     // Edgecase of username starting with team also works
-    expect(userTypeRouteRegExp.exec('/teampro/30')?.groups).toContain({
+    expect(userTypeRouteMatch('/teampro/30')?.params).toContain({
 			user: 'teampro',
 			type: '30'
 		})
 
-    expect(userTypeRouteRegExp.exec('/teampro+pro/30')?.groups).toContain({
+    // Edgecase of username starting with team also works
+    expect(userTypeRouteMatch('/workflowteam/30')?.params).toContain({
+      user: 'workflowteam',
+      type: '30'
+    })
+
+    expect(userTypeRouteMatch('/teampro+pro/30')?.params).toContain({
       user: 'teampro+pro',
       type: '30'
     })
 
-    expect(userTypeRouteRegExp.exec('/teampro+pro/book')).toEqual(null)
+    expect(userTypeRouteMatch('/teampro+pro/book')).toEqual(false)
 
     // Because /book doesn't have a corresponding new-booker route.
-    expect(userTypeRouteRegExp.exec('/free/book')).toEqual(null)
+    expect(userTypeRouteMatch('/free/book')).toEqual(false)
 
     // Because /booked is a normal event name
-    expect(userTypeRouteRegExp.exec('/free/booked')?.groups).toEqual({
+    expect(userTypeRouteMatch('/free/booked')?.params).toEqual({
 			user: 'free',
 			type: 'booked'
 		})
     
 
-    expect(embedUserTypeRouteRegExp.exec('/free/30/embed')?.groups).toContain({
+    expect(embedUserTypeRouteMatch('/free/30/embed')?.params).toContain({
       user: 'free',
       type:'30'
     })
 
     // Edgecase of username starting with team also works
-    expect(embedUserTypeRouteRegExp.exec('/teampro/30/embed')?.groups).toContain({
+    expect(embedUserTypeRouteMatch('/teampro/30/embed')?.params).toContain({
 			user: 'teampro',
 			type: '30'
 		})
 
-    expect(teamTypeRouteRegExp.exec('/team/seeded/30')?.groups).toContain({
+    expect(teamTypeRouteMatch('/team/seeded/30')?.params).toContain({
 			slug: 'seeded',
 			type: '30'
 		})
     
     // Because /book doesn't have a corresponding new-booker route.
-    expect(teamTypeRouteRegExp.exec('/team/seeded/book')).toEqual(null)
+    expect(teamTypeRouteMatch('/team/seeded/book')).toEqual(false)
 
-    expect(teamTypeRouteRegExp.exec('/team/seeded/30/embed')).toEqual(null)
+    expect(teamTypeRouteMatch('/team/seeded/30/embed')).toEqual(false)
 
-    expect(embedTeamTypeRouteRegExp.exec('/team/seeded/30/embed')?.groups).toContain({
+    expect(embedTeamTypeRouteMatch('/team/seeded/30/embed')?.params).toContain({
       slug: 'seeded',
       type:'30'
     })
 
-		expect(privateLinkRouteRegExp.exec('/d/3v4s321CXRJZx5TFxkpPvd/30min')?.groups).toContain({
+		expect(privateLinkRouteMatch('/d/3v4s321CXRJZx5TFxkpPvd/30min')?.params).toContain({
```

Comment:
Any reason why this has different identation?

Reply:
Not intentional. Prettier is disabled for test files and it's  just not fixing things.

Reply:
Fixed